consensus/types/src/light_client_bootstrap.rs

```diff
@@ -21,17 +21,15 @@ pub struct LightClientBootstrap<T: EthSpec> {
 }
 
 impl<T: EthSpec> LightClientBootstrap<T> {
-    pub fn from_beacon_state(beacon_state: BeaconState<T>) -> Result<Self, Error> {
+    pub fn from_beacon_state(mut beacon_state: BeaconState<T>) -> Result<Self, Error> {
```

Comment:
we probably want this function to just take a mutable reference to the state, rather than consuming it (which would require moving/cloning it)

```suggestion
    pub fn from_beacon_state(beacon_state: &mut BeaconState<T>) -> Result<Self, Error> {
```